jdat_notebooks/README.md

```diff
@@ -72,17 +72,17 @@ simple function that just did ``spec = simulate_jwst_spectrum(a, b)``
 thereby providing guidance for where specific development would simplify the workflow.
 
 If a notebook is freshly created in this form, the author can follow the "Procedure to submit a notebook as a Pull Request"
-(found at the end of this document), skipping the Notebook Concept step.  
+(found at the end of this document), skipping the Draft Stage step.  
 
-If the notebook was already created in the Concept Notebook step and the "Procedure to submit a notebook as a Pull Request"
+If the notebook was already created in the Draft Stage step and the "Procedure to submit a notebook as a Pull Request"
 has already been followed, the author should just create a new branch to modify the existing code and then create 
 a new Pull Request with the changes once they are ready.
 
-In either case, the title (but not filename) of the notebook should begin with "Draft:" to indicate the 
-notebook is in the draft stage.
+In either case, the title (but not filename) of the notebook should begin with 
+"Baseline:" to indicate the notebook is in the Baseline Stage.
```

Comment:
We haven't necessarily been doing this.  Do we want to stick with this, or should we change to the title just being the title and categorizing it using github or Jira?

(If we *do* still like this we should probably add it to the technical review to make sure the title is right...)

Reply:
I would like to have the title in the notebook (not in the file) to reflect the status, so "Baseline: blablabla...". Agree, we should add this to the technical review!